src/components/shared/StepWidget.tsx

```diff
@@ -0,0 +1,52 @@
+import React, { useState } from 'react';
+import { TiTick } from 'react-icons/ti';
+import '../../assets/styles/components/stepper.scss';
```

Comment:
I don't think you have to import this here. You can import all styles from one single file, the `_global.scss` and that will just mount your styles along with the rest. 

Take a look at that file, you'll get the context.